resources/js/components/station-autocomplete.js

```diff
@@ -40,7 +40,7 @@ const popularStations = [
     function fetchStations() {
         if (input.value.length < 2) return;
 
-        fetch("/transport/train/autocomplete/" + encodeURI(input.value))
+        fetch("/admin/transport/train/autocomplete/" + encodeURI(input.value))
```

Comment:
is this temporary?

Reply:
Was stupid here. Thought this endpoint is only used in the admin pages but it's still used in the event page.